tuxbot/main.go

```diff
@@ -7,13 +7,42 @@ import (
 	"bytes"
 	"log"
 	"os"
+	"os/exec"
 
 	"github.com/keybase/slackbot"
 	"github.com/keybase/slackbot/cli"
+	"github.com/nlopes/slack"
 	"gopkg.in/alecthomas/kingpin.v2"
 )
 
-func kingpinTuxbotHandler(args []string) (string, error) {
+var token string
```

Comment:
I think the Right Thing to do here is going to be to add something additional into the signature for all these commands, so they're given what they need to make API requests. It could be just the `api` object itself, or it could be something more abstract?